Content.Server/BluespaceHarvester/BluespaceHarvesterBundleSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Shared.Destructible;
+using Content.Shared.Storage.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.BluespaceHarvester;
+
+public sealed class BluespaceHarvesterBundleSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<BluespaceHarvesterBundleComponent, StorageBeforeOpenEvent>(OnOpen);
+        SubscribeLocalEvent<BluespaceHarvesterBundleComponent, DestructionEventArgs>(OnDestruction);
+    }
+
+    private void OnOpen(Entity<BluespaceHarvesterBundleComponent> bundle, ref StorageBeforeOpenEvent args)
+    {
+        CreateLoot(bundle);
```

Comment:
I take it the main purpose here is to be able to add spawners inside crates. Maybe it makes sense not to create new components and systems, but to edit the standard spawners so that they work in crates?